metricbeat/module/mongodb/status/data.go

```diff
@@ -0,0 +1,241 @@
+package status
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/elastic/beats/libbeat/common"
+	"github.com/elastic/beats/libbeat/logp"
+)
+
+func eventMapping(status map[string]interface{}) common.MapStr {
+	errs := map[string]error{}
+	event := common.MapStr{
+		"version": mustBeString("version", status, errs),
+		"uptime": common.MapStr{
+			"ms": mustBeInteger("uptimeMillis", status, errs),
+		},
+		"local_time":         mustBeTime("localTime", status, errs),
+		"write_backs_queued": mustBeBool("writeBacksQueued", status, errs),
+	}
+	removeErroredKeys(event, errs)
```

Comment:
Right, IMO kibana does a relatively good job for when there are no keys. I would be more worried if we fill it with zeros, as that might hide real issues and we lose users' trust.